be/src/exprs/runtime_filter.cpp

```diff
@@ -78,8 +78,10 @@ size_t SimdBlockFilter::serialize(uint8_t* data) const {
     const size_t alloc_size = get_alloc_size();
     int32_t data_size = alloc_size;
     SIMD_BF_COPY_FIELD(data_size);
-    memcpy(data + offset, _directory, data_size);
-    offset += data_size;
+    if (LIKELY(data_size > 0)) {
```

Comment:
move this check forward to L:80?

Reply:
> move this check forward to L:80?

we can't do this because we still need to copy data_size although it equals 0.